src/util/extend.js

```diff
@@ -7,10 +7,6 @@
  * @return {Object} Merged object
  */
 export default function extend(dest, ...sources) {
-    sources.forEach(source => {
-        Object.keys(source).forEach(key => {
-            dest[key] = source[key];
-        });
-    });
-    return dest;
+    console.warn('util.extend is depreciated; use Object.assign');
```

Comment:
typo: it's `deprecated`

Reply:
don't remove the method content, it's deprecated, not removed.

Reply:
Typo fixed; thanks!

> don't remove the method content, it's deprecated, not removed.

I'm only changing the internal implementation, as @katspaugh suggested.